.github/workflows/binary-builds.yml

```diff
@@ -115,10 +121,16 @@ jobs:
             cmd: |
               # Prepare workspace
               Remove-Item ci -Recurse -Force
+              Remove-Item test -Recurse -Force
+              Remove-Item devenv.* -Force
+              Remove-Item -Path ADVANCED.md,pyproject.toml,renovate.json,uv.lock -Force
               Remove-Item contrib -Recurse -Force
               Remove-Item tools_config -Recurse -Force
               npm install --omit=dev --no-package-lock --no-audit --no-fund --no-progress
 
+              # Generate sbom (imprecise)
+              node bin/cdxgen.js -t js --exclude "**/test/**" -o sbom-postbuild-js.cdx.json --include-formulation
```

Comment:
Since we build Windows binaries using npm and without a lock file, the generated SBOM is slightly imprecise since it will be based on the pnpm lockfile.